client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/style.scss

```diff
@@ -1,4 +1,6 @@
 @import '@wordpress/base-styles/variables';
+@import '@wordpress/base-styles/_breakpoints.scss';
```

Comment:
We can just use `@import '@wordpress/base-styles/breakpoints` and `@import '@wordpress/base-styles/mixins'`